process/factory/hyperblock_processor_factory.go

```diff
@@ -24,5 +25,6 @@ func CreateHyperBlockProcessor(args *ArgsHyperBlockProcessor) (covalent.HyperBlo
 	}
 
 	shardBlocksHandler := shardBlocks.NewShardBlocksProcessor()
-	return process.NewHyperBlockProcessor(transactionsHandler, shardBlocksHandler)
+	epochStartHandler := epochStart.NewEpochStartInfoProcessor()
```

Comment:
epochStartInfoProcessorHandler

Reply:
Renamed it to `epochStartInfoHandler`.(subject to change depending on the reply above)